pkg/email/templates/added_as_collaborator_body.template

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+   <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+      <title>Content</title>
+      <style type="text/css"> * {margin:0; padding:0; text-indent:0; }
+         h1 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 39pt; }
+         .s1 { color: #707579; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
+         p { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; margin:0pt; }
+         h2 { color: #005BA0; font-family:Arial, sans-serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 15pt; }
+         .a { color: #005BA0; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
+         .s2 { color: #1A1A1A; font-family:Arial, sans-serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 15.5pt; }
+      </style>
+   </head>
+   <body>
+      <h1 style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">MINT</h1>
+      <p class="s1" style="padding-top: 11pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">The Model Innovation Tool</p>
+      <p style="text-indent: 0pt;text-align: left;"><br/></p>
+      <p style="padding-top: 14pt;padding-left: 5pt;text-indent: 0pt;line-height: 140%;text-align: left;">You&#39;ve been added as a collaborator on {{.ModelName}} in MINT.</p>
+      <p style="text-indent: 0pt;text-align: left;"><br/></p>
+      <h2 style="padding-left: 5pt;text-indent: 0pt;text-align: left;"><a href="{{.ClientAddress}}/models/{{.ModelID}}/task-list">Start collaborating now</a></h2>
+      <p style="text-indent: 0pt;text-align: left;"><br/></p>
+      <p style="padding-top: 12pt;padding-left: 5pt;text-indent: 0pt;line-height: 140%;text-align: left;"><a href="mailto:MINTTeam@cms.hhs.gov" class="s2" target="_blank">If you have not already been given access to MINT or if you cannot log in with your EUA credentials, please contact the Model Assessment Team at </a><a href="mailto:MINTTeam@cms.hhs.gov" class="a" target="_blank">MINTTeam@cms.hhs.gov</a><a href="mailto:MINTTeam@cms.hhs.gov" class="s2" target="_blank">.</a></p>

```

Comment:
This line appears to link `mailto:` twice, so the entire line is clickable. We should probably remove the first one and just have the email itself be hyperlinked

Reply:
Good catch. Cleaned up, testing now.